lib/scripts/utils.sh

```diff
@@ -20,18 +20,73 @@ export TRACE_AGENT_CMD="./trace-agent --config dist/datadog.yaml --pid run/trace
 export DOGSTATSD_PIDFILE="${DATADOG_DIR}/run/dogstatsd.pid"
 export DOGSTATSD_CMD="./dogstatsd start --cfgpath dist/"
 
-log_info() {
-  log_message "$0" "$$" "$@" "INFO"
+# the logging level of the update_agent_script.log file
+export DD_UPDATE_SCRIPT_LOG_LEVEL="${DD_UPDATE_SCRIPT_LOG_LEVEL:-"INFO"}"
+
+dd_export_env() {
+  local env_file="$1"
+
+  if [ -n "${DD_ENABLE_CAPI_METADATA_COLLECTION:-}" ]; then
```

Comment:
```suggestion
  if [ -n "${DD_ENABLE_CAPI_METADATA_COLLECTION}" ]; then
```
Let's remove those, as we don't use `set -u`, it should not be needed.